src/microbe_stage/systems/EngulfingSystem.cs

```diff
@@ -186,6 +187,19 @@ protected override void Update(float delta, in Entity entity)
             return;
         }
 
+        // Not full anymore tutorial (end trigger for engulfment full tutorial)
+        if (entity.Has<PlayerMarker>())
```

Comment:
I'd just rework the the system to always just send the event of not full if there exists a listener for it. Because right now `wasFullAlready` would need to be saved in saves, and the naming of the callback is not clear that it would only trigger for the player.